src/components/user/userRoutes.js

```diff
@@ -1,17 +1,15 @@
-import express from 'express';
+import { Router } from 'express';
 
-import * as userController from './userController.js';
-import * as authController from '../auth/authController.js';
+import { loginUser, protect, signUp } from '../auth/authController.js';
+import { getUsers, updateUser } from './userController.js';
 
-const router = express.Router();
+const router = Router();
 
 // USER ROUTES
-router.route('/').get(authController.protect, userController.getUsers);
+router.route('/').get(protect, getUsers);
+router.route('/update-me').patch(protect, updateUser);
```

Comment:
If a route has same http methods like GET,POST then instead of using "router.route('/url').post(funcA)" use "router.post('/url',funcA)"